lib/OnboardingMessageProvider.jsm

```diff
@@ -480,6 +480,8 @@ const ONBOARDING_MESSAGES = async () => [
     id: `WHATS_NEW_BADGE_${FIREFOX_VERSION}`,
     template: "toolbar_badge",
     content: {
+      // delay: 5 * 3600 * 1000,
+      delay: 5000,
```

Comment:
Just for testing, will revert after review.

Reply:
Hmm, when you put the old delay back in, it's going to be hard to test from devtools. We'll need to find a workaround probably.

Reply:
I was thinking we move all our test messages to staging, it's easy to switch using the RS-devtools and we can set values that make it easier to test.